mirage/dark/dark_prep.py

```diff
@@ -34,14 +34,12 @@
 import numpy as np
 from astropy.io import fits, ascii
 
-
+import mirage
 from mirage.utils import read_fits, utils, siaf_interface
 from mirage.utils.file_splitting import find_file_splits
 from mirage.reference_files import crds_tools
 
-from mirage import version
-
-MIRAGE_VERSION = version.__version__
+MIRAGE_VERSION = mirage.__version__
```

Comment:
You could also do something like
```
from mirage import __version__ as mirage_version
MIRAGE_VERSION = mirage_version
```
or dispense with the global variables and just use `mirage.__version__` everywhere it is needed.

Reply:
Good idea. I switched everything to use `mirage.__version__`